grpc-runtime/src/main/java/io/micronaut/grpc/server/GrpcEmbeddedServer.java

```diff
@@ -58,7 +58,7 @@
  */
 @Singleton
 @Secondary
-@Requires(classes = ServerBuilder.class)
+@Requires(classes = ServerBuilder.class, condition = GrpcServerEnabledCondition.class)
```

Comment:
This can simplified to `@Requires(property="grpc.server.enabled", value=StringUtils.TRUE, defaultValue=StringUtils.TRUE)` instead of using a custom condition.

Reply:
I add no idea, and it is indeed much better, I have updated my commit accordingly,